src/test/scala/chiselTests/Vec.scala

```diff
@@ -308,4 +308,19 @@ class VecSpec extends ChiselPropSpec with Utils {
         }}
     }
   }
+
+  property("Vec connections should emit FIRRTL bulk connects when possible") {
```

Comment:
should we include some of the "when not possible" cases in the tests?

Reply:
Yes we should

Reply:
Some cases we should test:

1. `import chisel3._` Bundle that instantiates an `import Chisel._` Bundle that relies on the Stringly-typed connections for the connect to be legal, this should not emit a bulk connect. [Some inspiration](https://scastie.scala-lang.org/wy301om6RQii8u6iSe1DRw) -- this example is a little weird because I'm surprised that `out.buzz.foo <= in.buzz.foo` in the emitted CHIRRTL uses `<=` instead of `<-`)
2. A case where `<>` will magically work but the FIRRTL types would be wrong for `<=`, eg. https://scastie.scala-lang.org/9xV1AAU0SFq1UvQLVbGFFw -- `enq <= deq` is illegal while Chisel can do the connecting by splitting it up.

Reply:
Similar principles apply in Vecs, in fact you could make Vecs of these Bundles